_docs/migrate_module.md

```diff
@@ -16,8 +16,10 @@ You will have someone by your side in this process. The general flow is to…
 * Add the module to our [modulesync setup](managed_modules).
 * Add the module to our [plumbing repository](plumbing)(handles travis secrets).
 * Ask an admin to add the `collaborators` team to the module's `Collaborators & Teams` 'Teams' list with `Write` permissions (e.g. [https://github.com/voxpupuli/puppet-gitlab/settings/collaboration](https://github.com/voxpupuli/puppet-gitlab/settings/collaboration))).
-* Execute modulesync for this module.
+* Execute [modulesync](msync) for this module.
 * Create a Jira issue at [https://tickets.puppetlabs.com](https://tickets.puppetlabs.com) and ask to deprecate the old module (and approve the new one if the old one was approved as well).
+* Ensure that there isn't a `CONTRIBUTING.md` file in the docroot. Our modulesync will place one at `.github/CONTRIBUTING.md`. Please enhance [our existing template](template) if the version in the docroot contains useful parts.
```

Comment:
If we care about this, we should set it in config_defaults.yaml to let msync ensure it's absent.